src/pages/_document.tsx

```diff
@@ -0,0 +1,24 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document';
```

Comment:
I did not know about next/document. This can be used to alter what's in the body tag. Neat!